src/__test__/App.test.js

```diff
@@ -1,10 +1,13 @@
-
-
 import React from 'react';
-import App from '../App.jsx';
+import App from '../App';
+import PageContainer from '../component/PageContainer';
```

Comment:
I think It's better to test the child component without importing it, and just by using its name.